simulator/func_sim/rf/rf.h

```diff
@@ -15,17 +15,18 @@ class RF
 {
     using FuncInstr = typename ISA::FuncInstr;
     using Register = typename ISA::Register;
+	using RegisterUInt = typename ISA::RegisterUInt;
```

Comment:
Please replace \t with 4 spaces here and everywhere

Reply:
Sorry, forgot to fix vimrc file...ashamed(